src/mongocxx/test/spec/monitoring.cpp

```diff
@@ -147,15 +210,21 @@ std::string apm_checker::to_string(event e) {
             return "killCursors";
         case apm_checker::event::get_more:
             return "getMore";
+        case apm_checker::event::configure_fail_point:
+            return "configureFailPoint";
     }
 }
 
 void apm_checker::set_ignore_command_monitoring_event(event e) {
     this->_ignore.push_back(to_string(e));
 }
 
-void apm_checker::clear() {
+void apm_checker::clear_events() {
```

Comment:
This allows events to be cleared just before running operations without destroying the list of commands we plan to ignore.